server/models/Pet.js

```diff
@@ -3,19 +3,31 @@ import pg from "pg"
 const pool = new pg.Pool({ connectionString: "postgres://postgres:password@localhost:5432/new_family_member_development" })
 
 class Pet {
-    constructor({ id, name, img_url, imgUrl, age, vaccination_status, vaccinationStatus,
-        adoption_story, adoptionStory, available_for_adoption,
-        availableForAdoption, pet_type_id, petTypeId }) {
-        this.id = id
-        this.name = name
-        this.imgUrl = imgUrl || img_url
-        this.age = age
-        this.vaccinationStatus = vaccinationStatus || vaccination_status
-        this.adoptionStory = adoptionStory || adoption_story
-        this.availableForAdoption = availableForAdoption || available_for_adoption
-        this.petTypeId = petTypeId || pet_type_id
-    }
+  constructor ({id, petName, name, age, petAge, img_url, imgUrl, vaccination_status = false, vaccinationStatus = false,
+          adoption_story = "N/A", adoptionStory = "N/A", available_for_adoption = true, 
+          availableForAdoption = true, pet_type_id, petTypeId}){
+    this.id = id
+    this.name = petName || name
```

Comment:
ideally, rather than account for this name coming in with a totally different variable name and over-extending the signature of the Pet's constructor, we could handle for this in our controller/router like so:
```
const newPet = new Pet({...req.body, name: req.body.petName})
```
Same with petAge!